packages/core/src/compiler/Style.ts

```diff
@@ -2128,10 +2130,21 @@ const findPathsExpr = <T>(expr: Expr<T>): Path<T>[] => {
     case "Vary": {
       return [];
     }
-    case "CompApp":
+    case "CompApp": {
+      return expr.args.flatMap(findPathsExpr);
+    }
     case "ConstrFn":
     case "ObjFn": {
-      return expr.args.flatMap(findPathsExpr);
+      const body = expr.body;
+      switch (body.tag) {
+        case "FunctionCall": {
+          return body.args.flatMap(findPathsExpr);
+        }
+        case "InlineComparison": {
+          return [body.arg1, body.arg2].flatMap(findPathsExpr);
+        }
+      }
+      break;
```

Comment:
I see that our particular cocktail of TypeScript and ESLint causes you to have to insert a dead `break;` here due to nested `switch` statements; you could instead consider pulling this case out into its own function, to avoid the dead code.

Reply:
I decided to use `if` statements instead